test/array.js

```diff
@@ -8,180 +8,205 @@
 'use strict';
 
 require('should');
-var utils = require('..')._;
+var utils = require('../index').array;
 
```

Comment:
before one yea this failing on my previous ubuntu, so since then I always add it.